src/content/add-to-app/ios/project-setup.md

```diff
@@ -152,108 +156,103 @@ some/path/
     └── Podfile
 ```
 
-If your existing application (`MyApp`) doesn't
-already have a Podfile, run `pod init` in the  
-`MyApp` directory to create one. 
-You can find more details on using 
-CocoaPods in the [CocoaPods getting started guide][].
+#### Update your Podfile
 
-<ol>
-<li>
+1. If your existing application (`MyApp`) doesn't already have a Podfile,
+   use the `pod init` command in the `MyApp` directory to create one.
 
-Add the following lines to your `Podfile`:
+   To learn more about CocoaPods,
+   consult the [CocoaPods getting started guide][].
 
-```ruby title="MyApp/Podfile"
-flutter_application_path = '../my_flutter'
-load File.join(flutter_application_path, '.ios', 'Flutter', 'podhelper.rb')
-```
+1. Add the following lines to your `Podfile`.
 
-</li>
+   ```ruby title="MyApp/Podfile"
+   flutter_application_path = '../my_flutter'
+   load File.join(flutter_application_path, '.ios', 'Flutter', 'podhelper.rb')
+   ```
 
-<li>
+1. For each [Podfile target][] that needs to embed Flutter,
+   add a call to `install_all_flutter_pods(flutter_application_path)`.
 
-For each [Podfile target][] that needs to
-embed Flutter, call `install_all_flutter_pods(flutter_application_path)`.
+   ```ruby title="MyApp/Podfile"
+   target 'MyApp' do
+     install_all_flutter_pods(flutter_application_path)
+   end
+   ```
 
-```ruby title="MyApp/Podfile"
-target 'MyApp' do
-  install_all_flutter_pods(flutter_application_path)
-end
-```
+1. In the `Podfile`'s `post_install` block,
+   add a call to `flutter_post_install(installer)`.
 
-</li>
+   ```ruby title="MyApp/Podfile"
+   post_install do |installer|
+     flutter_post_install(installer) if defined?(flutter_post_install)
+   end
+   ```
 
-<li>
+   The `flutter_post_install` method adds build settings to support
```

Comment:
A new `Podfile` does not have a `post_install` block. Should we add a note on this scenario? Perhaps:

```suggestion
   :::note
   If your `Podfile` does not have a `post_install` block yet, add this code to the bottom of your `Podfile`.
   :::
```